pom.xml

```diff
@@ -21,11 +21,15 @@
   </licenses>
   <properties>
     <powermock.version>1.6.2</powermock.version>
-    <jenkins.version>1.580.1</jenkins.version>
-    <java.level>6</java.level>
+    <java.level>7</java.level>
   </properties>
   <dependencies>
     <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>display-url-api</artifactId>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
Depending on a snapshot? Has it even been released yet? If not, then this PR can't go through until it has.